### ISSUE TITLE
Size of column in BO is not the same in all page

### ISSUE THREAD

**Issue by sarahdib** [PrestaShop/PrestaShop#23015]
#### Describe the bug

Size of column in BO is not the same in all page

#### Steps to Reproduce

1. Go to admin -> preference general & order settings 
2. See error

**Screenshots**

<img width="1440" alt="Capture d’écran 2021-01-27 à 14 53 40" src="https://user-images.githubusercontent.com/3034677/106002707-bb225780-60b1-11eb-92f4-e8d0143905f0.png">

<img width="1440" alt="Capture d’écran 2021-01-27 à 14 53 49" src="https://user-images.githubusercontent.com/3034677/106002718-be1d4800-60b1-11eb-82e9-8c800e26ee88.png">

#### Additional information
* PrestaShop version: develop
* PHP version: 7.1

**Comment by MatShir** [comment]
Waiting for @prestascott @TristanLDD response.

**Comment by prestascott** [comment]
Hey everyone,

In fact, containers in the bo have 3 different sizes:
- 100 %
- 80 %
- 60 % (approximately)

It may seems weird for our users and they can identify these 3 different sizes as a display bug. 
This is why I suggest to keep only 1 size, which is the 100 %. I already discussed with @NeOMakinG & @TristanLDD and we're okay with this change. 

Although, we need to create a common rule about the grid system including the number of columns in the container to keep consistency. According to ergonomics rules, limit line length should be between 60 and 90 characters, long lines of text can confuse the eyes. 

In my design, I made sure that the body text in a line doesn't exceed 90 characters. 

**Here is my design proposition with the grid (1440 px):**
![Capture d’écran 2021-02-11 à 16 24 22](https://user-images.githubusercontent.com/54888056/107658885-e0ed5600-6c86-11eb-8c51-31703bf86e41.png)

**And without grid (1440 px):**
![Group 88](https://user-images.githubusercontent.com/54888056/107658724-b7ccc580-6c86-11eb-8b5e-78cd6de7d0e2.png)

**Behaviors:**
- Screens > 1440 px: Only the first and the last column get larger, so contents have a fix width and users avoid getting a torticollis while reading.  
- Screens < 1440 px: Only the first and the last column get smaller. Contents have a fix until they reach their breakpoints.

Wdyt?
